packages/@expo/cli/src/start/server/metro/__tests__/MetroBundlerDevServer-test.ts

```diff
@@ -52,7 +52,7 @@ beforeEach(() => {
 async function getStartedDevServer(options: Partial<BundlerStartOptions> = {}) {
   const devServer = new MetroBundlerDevServer(
     '/',
-    getPlatformBundlers({ web: { bundler: 'metro' } })
+    getPlatformBundlers(__dirname, { web: { bundler: 'metro' } })
```

Comment:
This is incorrect, but the `projectRoot` value is not used as the `web.bundler` is specified.

Reply:
This just needs to align with what's sent to the getConfig method, which is mocked out. A "/" would probably be less confusing.